scripts/LuaJIT.sh

```diff
@@ -1,8 +1,16 @@
 #!/bin/bash -e
-ver=2.1.0-beta3
+
+repo=https://luajit.org/git/luajit.git
+rev=91bc6b8ad1f373c1ce9003dc024b2e21fad0e444
 
 download () {
-	get_tar_archive LuaJIT "https://luajit.org/download/LuaJIT-${ver}.tar.gz"
+	if [ ! -d LuaJIT/.git ]; then
+		rm -rf LuaJIT
+		git clone $repo LuaJIT
+		pushd LuaJIT
+		git checkout $rev
+		popd
+	fi
```

Comment:
```suggestion
	[ -d LuaJIT ] || git clone $repo-b $rev LuaJIT
```

Reply:
I couldn't find a way to get clone to work with hashes:
```
$ git clone "https://luajit.org/git/luajit.git" -b 91bc6b8ad1f373c1ce9003dc024b2e21fad0e444 LuaJIT
Cloning into 'LuaJIT'...
fatal: Remote branch 91bc6b8ad1f373c1ce9003dc024b2e21fad0e444 not found in upstream origin
```

Reply:
uhh right